test/powershell/engine/HelpSystem.OnlineHelp.Tests.ps1

```diff
@@ -1,20 +1,30 @@
 Describe 'Online help tests for PowerShell Core Cmdlets' -Tags "CI" {
 
     BeforeAll {
+
+        $skipTest = [System.Management.Automation.Platform]::IsLinux
```

Comment:
Why are we skipping linux?
And with `$skipTest`, the test will run on OSX. Are you intend to run this test on OSX platform?